ecl/hql/hqlexpr.hpp

```diff
@@ -1022,7 +1022,6 @@ class HQL_API HqlLookupContext
     inline bool syntaxChecking() const { return parseCtx.isSyntaxChecking(); }
     inline bool regenerateCache() const { return parseCtx.regenerateCache; }
     inline bool hasCacheLocation() const { return parseCtx.hasCacheLocation();}
-    inline bool syntaxCheckingUsingCache() const { return parseCtx.isSyntaxChecking() && parseCtx.hasCacheLocation(); }
```

Comment:
I've removed this function because after commoning up the code below this function is used just once.